Fix init script for lintian error
Index: laptop-mode-tools/etc/init.d/laptop-mode
===================================================================
--- laptop-mode-tools.orig/etc/init.d/laptop-mode	2014-06-18 19:46:27.208495742 +0530
+++ laptop-mode-tools/etc/init.d/laptop-mode	2014-07-19 16:50:16.465969396 +0530
@@ -8,7 +8,6 @@
 
 ### BEGIN INIT INFO
 # Provides:          laptop-mode
-# Should-Start:      $all
 # Required-Start:    $remote_fs
 # Required-Stop:     $remote_fs
 # Default-Start:     2 3 4 5
