.desktop file for gui config
Index: laptop-mode-tools/gui/laptop-mode-tools.desktop
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ laptop-mode-tools/gui/laptop-mode-tools.desktop	2014-06-19 12:12:45.154704322 +0530
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=Laptop Mode Tools Configuration
+GenericName=Laptop Mode Tools Configuration
+Exec=/usr/sbin/lmt-config-gui
+Categories=System;
+Comment=Enable/disable Laptop Mode Tools settings
+Keywords=Preferences;
