Fix spam of messages on /dev/console. Fixed upstream
Index: laptop-mode-tools/etc/systemd/laptop-mode.service
===================================================================
--- laptop-mode-tools.orig/etc/systemd/laptop-mode.service	2015-09-03 16:31:16.625039558 +0530
+++ laptop-mode-tools/etc/systemd/laptop-mode.service	2015-09-03 16:33:12.492038939 +0530
@@ -10,8 +10,8 @@
 ExecStop=@/usr/sbin/laptop_mode init stop
 ExecStopPost=@/bin/rm -f /var/run/laptop-mode-tools/enabled
 ExecReload=@/usr/sbin/laptop_mode auto
-StandardOutput=tty
-StandardError=tty
+StandardOutput=journal
+StandardError=journal
 
 [Install]
 WantedBy=multi-user.target
Index: laptop-mode-tools/etc/systemd/lmt-poll.service
===================================================================
--- laptop-mode-tools.orig/etc/systemd/lmt-poll.service	2015-09-03 16:31:11.651868845 +0530
+++ laptop-mode-tools/etc/systemd/lmt-poll.service	2015-09-03 16:34:29.199852330 +0530
@@ -6,10 +6,10 @@
 [Service]
 Type=oneshot
 RemainAfterExit=no
-ExecStart=@/usr/sbin/laptop_mode auto
-ExecReload=@/usr/sbin/laptop_mode auto
-StandardOutput=tty
-StandardError=tty
+ExecStart=/lib/udev/lmt-udev
+ExecReload=/lib/udev/lmt-udev
+StandardOutput=journal
+StandardError=journal
 
 [Install]
 WantedBy=multi-user.target
Index: laptop-mode-tools/usr/sbin/laptop_mode
===================================================================
--- laptop-mode-tools.orig/usr/sbin/laptop_mode	2015-09-03 16:31:11.655869837 +0530
+++ laptop-mode-tools/usr/sbin/laptop_mode	2015-09-03 16:35:14.442780387 +0530
@@ -190,12 +190,12 @@
 	fi
 
 	if [ "$1" = "VERBOSE" ]; then
-		$LM_VERBOSE &&  echo "$2" >>/dev/fd/2;
+		$LM_VERBOSE &&  echo "$2" >&2;
 	elif [ "$1" = "ERR" ]; then
-		echo "$2" >>/dev/fd/2;
+		echo "$2" >&2;
 	else
 		# Message of type MSG and STATUS can go to stdout.
-		echo "$2" >>/dev/fd/1;
+		echo "$2" >&1;
 	fi
 }
 
